src/encoded/static/components/static-pages/ErrorPage.js

```diff
@@ -28,7 +31,9 @@ export default class ErrorPage extends React.PureComponent {
                     </h5>
                     <h5>Access is restricted to 4DN consortium members.</h5>
                     <h5>
-                        <a href="mailto:4DN.DCIC.support@hms-dbmi.atlassian.net" className="link-underline-hover">
+                        <a
+                            href="mailto:4DN.DCIC.support@hms-dbmi.atlassian.net"
+                            className="link-underline-hover">
```

Comment:
While you're at this, it might be good to change this to point to the smaht dac atlassian support page. I think the email we have used in the past is: `smhelp@hms-dbmi.atlassian.net`